photon-core/src/main/java/org/photonvision/vision/pipeline/ObjectDetectionPipeline.java

```diff
@@ -113,10 +121,13 @@ protected void setPipeParamsImpl() {
     protected CVPipelineResult process(Frame frame, ObjectDetectionPipelineSettings settings) {
         long sumPipeNanosElapsed = 0;
 
+        CVPipeResult<CVMat> croppedFrame = cropPipe.run(frame.colorImage);
+        sumPipeNanosElapsed += croppedFrame.nanosElapsed;
+
         // ***************** change based on backend ***********************
 
         CVPipeResult<List<NeuralNetworkPipeResult>> rknnResult =
-                objectDetectorPipe.run(frame.colorImage);
+                objectDetectorPipe.run(croppedFrame.output);
```

Comment:
Same as other pipelines; deal with releasing croppedFrame, when necessary. Dito for colored shape and aruco etc